yt/frontends/cf_radial/data_structures.py

```diff
@@ -0,0 +1,173 @@
+"""
+CF Radial data structures
+
+
+
+"""
+
+import os
+import weakref
+
+import numpy as np
+
+from yt.data_objects.index_subobjects.grid_patch import AMRGridPatch
+from yt.data_objects.static_output import Dataset
+from yt.funcs import mylog
+from yt.geometry.grid_geometry_handler import GridIndex
+from yt.utilities.on_demand_imports import _xarray as xr
+
+from .fields import CFRadialFieldInfo
+
+
+class CFRadialGrid(AMRGridPatch):
+    _id_offset = 0
+
+    def __init__(self, id, index, level, dimensions):
+        super().__init__(id, filename=index.index_filename, index=index)
+        self.Parent = None
+        self.Children = []
+        self.Level = level
+        self.ActiveDimensions = dimensions
+
+    def __repr__(self):
+        return "CFRadialGrid_%04i (%s)" % (self.id, self.ActiveDimensions)
+
+
+class CFRadialHierarchy(GridIndex):
+    grid = CFRadialGrid
+
+    def __init__(self, ds, dataset_type="cf_radial"):
+        self.dataset_type = dataset_type
+        self.dataset = weakref.proxy(ds)
+        # our index file is the dataset itself:
+        self.index_filename = self.dataset.parameter_filename
+        self.directory = os.path.dirname(self.index_filename)
+        # float type for the simulation edges and must be float64 now
+        self.float_type = np.float64
+        super().__init__(ds, dataset_type)
+
+    def _initialize_state_variables(self):
+        super()._initialize_state_variables()
+
```

Comment:
what's the rationale for having this method ? in this state it's equivalent to not overriding the parent class' method
```suggestion
```

Reply:
Probably just leftovers from initially building out the frontend. I think it's ok to delete!